nupic/research/frameworks/pytorch/imagenet/imagenet_experiment.py

```diff
@@ -77,7 +74,7 @@ class ImagenetExperiment:
     def __init__(self):
         self.model = None
         self.optimizer = None
-        self.loss_function = None
+        self.loss_function_f = None
```

Comment:
Maybe you could use underscore to indicate this is a private member: `self._loss_function`